src/styles/conversions.ts

```diff
@@ -0,0 +1,141 @@
+/* @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {NumberNode, ZERO} from './parsers';
+
+/**
+ * Ensures that a given number is expressed in radians. If the number is already
+ * in radians, does nothing. If the value is in degrees, converts it to radians.
+ * If the value has no specified unit, the unit is assumed to be radians. If the
+ * value is not in radians or degrees, the value is resolved as 0 radians.
+ */
+export const degreesToRadians =
+    (numberNode: NumberNode, defaultRadians: number = 0): NumberNode => {
```

Comment:
Perhaps change defaultRadians to fallbackRadiansValue and put a comment matching the lengthToBaseMeters comment below?